ui/lib/core/addon/components/policy-example.hbs

```diff
@@ -0,0 +1,43 @@
+<div class="has-bottom-margin-s">
+  {{#if (eq @policyType "acl")}}
+    <p data-test-example-modal-text-acl>
+      ACL Policies are written in Hashicorp Configuration Language (
+      <ExternalLink @href="https://github.com/hashicorp/hcl">HCL</ExternalLink>
+      ) or JSON and describe which paths in Vault a user or machine is allowed to access. Here is an example policy:
+    </p>
+  {{else}}
+    <p class="has-bottom-margin-s" data-test-example-modal-text-rgp>
+      Role Governing Policies (RGPs) are tied to client tokens or identities which is similar to
+      <DocLink @path="/vault/tutorials/policies/policies">ACL policies</DocLink>. They use
+      <DocLink @path="/vault/docs/enterprise/sentinel">Sentinel</DocLink>
+      as a language framework to enable fine-grained policy decisions.
+    </p>
+    <p>
+      Here is an example policy that uses RGP to restrict access to the
+      <code class="tag is-marginless is-paddingless">admin</code>
+      policy such that a user named James or has the
```

Comment:
this wasn't in the designs, but can we add the last name and put the name in quotes here? Since we're referencing the name itself, not the person, I believe it should be in quotes 🤔  
```suggestion
      policy such that a user named "James Thomas" or has the
```